### ISSUE TITLE
 Paddle 2.5  Warning:: 0D Tensor cannot be used as 'Tensor.numpy()[0]' .问题如何解决

### ISSUE THREAD

**Issue by fzbird** [PaddlePaddle/Paddle#60086]
### 请提出你的问题 Please ask your question

我在使用paddle2.5时遇到以下错误：
 Warning:: 0D Tensor cannot be used as 'Tensor.numpy()[0]' . In order to avoid this problem, 0D Tensor will be changed to 1D numpy currently, but it's not correct and will be removed in release 2.6. For Tensor contain only one element, Please modify  'Tensor.numpy()[0]' to 'float(Tensor)' as soon as possible, otherwise 'Tensor.numpy()[0]' will raise error in release 2.6.

我的训练过程主要代码：
            #准备数据
            eatas, labels = data
            eatas = paddle.to_tensor(eatas)
            labels = paddle.to_tensor(labels)
            #前向计算的过程
            predicts = model(eatas)
            #计算损失，取一个批次样本损失的平均值
            loss = F.square_error_cost(predicts, labels) # 均方误差（常用于回归问题）
            avg_loss = paddle.mean(loss)
其中eatas类似为[[0.0002,0.0002,0.003],[0.002,0.0002,0.1],[0.003,0.004,0.005]]，labels类似为[[05],[0.2],[0]]，为何会出现上面的提示，我如何修改上述代码避免此错误提示？

**Comment by FlyingQianMM** [comment]
上面的提示是警告，应该是你的代码里有将0-D tensor使用'Tensor.numpy()[0]' 的操作，0-D tensor指的是shape为[]的tensor。
将操作从'Tensor.numpy()[0]'修改为'float(Tensor)'即可
